### PR TITLE
Added MD5 FIPS compatibility 

### PR DESCRIPTION
**Pull request by daveed12**

### CODE REVIEW

**Comment by MoodyMudit** [comment]
There is one more places where this needs to be done
https://github.com/hubblestack/hubble/blob/develop/hubblestack/modules/nebula_osquery.py#L1232

File: hubblestack/modules/network.py [hunk 1]
```diff
@@ -1560,7 +1561,10 @@ def connect(host, port=None, **kwargs):
         if proto == "udp":
             # Generate a random string of a
             # decent size to test UDP connection
-            md5h = hashlib.md5()
+            if isFipsEnabled:
+                md5h = hashlib.md5(usedforsecurity=False)
+            else:
+                md5h = hashlib.md5()
```
Comment (MoodyMudit) [inline hubblestack/modules/network.py:1567]: Need to make this change at https://github.com/hubblestack/hubble/blob/develop/hubblestack/modules/nebula_osquery.py#L1232
also

**Review by MoodyMudit — CHANGES_REQUESTED** [review]

File: hubblestack/modules/network.py (hunk 1 above)
Comment (daveed12) [reply hubblestack/modules/network.py:1567]: Good catch Mudit

**Review by MoodyMudit — APPROVED** [review]

**Comment by jettero** [comment]
this will pass tests after #968 merges; I had to fix tests mudit broke by re-enabling various config options

**Pull request merged**